src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import usersFromServer from '../../api/users';
+
+interface Props {
+  userId: number;
+}
+
+const getUser = (userId: number) => {
+  return usersFromServer.find((user) => user.id === userId);
+};
```

Comment:
Let's prepare the data in one place - in the App component.
The only way to pass some data in React using `props`.

Reply:
Could I create function helper `getUser` and use it in different components?